src/barplot.ts

```diff
@@ -50,11 +59,11 @@ export class BarPlot extends AVisInstance implements IVisInstance {
   private readonly $node: d3.Selection<BarPlot>;
 
   private xscale: d3.scale.Linear<number, number>;
-  private yscale: d3.scale.Linear<number, number>;
 
   constructor(public readonly data: INumericalVector, parent: Element, options: IBarPlotOptions = {}) {
     super();
     mixin(this.options, options);
+    this.options.initialScale = this.options.heightTo/(this.data.dim[0] * this.options.heighti);
```

Comment:
Height of row. It was defined in original barplot code. Should I rename it for better readability?